lib/meilisearch.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'meilisearch/version'
 require 'meilisearch/utils'
+require 'meilisearch/task'
```

Comment:
Would be indeed better
However
- If I forget the file, the test cannot run
- I will not test it and add in this PR since it's not really related. It can be a job for a refactor. Or I can open an issue :)

Reply:
Yeeah you're right, I was relating this issue https://github.com/meilisearch/meilisearch-ruby/pull/270 but this only happened after the release, so we will not have this problem because the test will not run indeed.

The only improvement here would be related to not need to require the file manually 🤔 (is not worth by now I think)

But I definitely agree with you about being another issue/PR 💇